broker/restapisrv.go

```diff
@@ -62,35 +60,21 @@ func (apisrv *RestApiSrv) Start(cfg *Config, broker *ThinBroker) {
 		rest.Delete("/v2/subscription/#sid", apisrv.deletev2Subscription),
 
 		//NGSI-LD APIs
-
-		//create and update
+		rest.Post("/ngsi-ld/v1/notifyContext/", broker.NotifyLdContext),
 		rest.Post("/ngsi-ld/v1/entities/", broker.LDCreateEntity),
+		rest.Get("/ngsi-ld/v1/entities/#eid", apisrv.LDGetEntity),
+		rest.Get("/ngsi-ld/v1/entities", apisrv.GetQueryParamsEntities),
 		rest.Post("/ngsi-ld/v1/entities/#eid/attrs", broker.LDAppendEntityAttributes),
 		rest.Patch("/ngsi-ld/v1/entities/#eid/attrs", broker.LDUpdateEntityAttributes),
 		rest.Patch("/ngsi-ld/v1/entities/#eid/attrs/#attr", broker.LDUpdateEntityByAttribute),
-
-		//query
-		rest.Get("/ngsi-ld/v1/entities/#eid", apisrv.LDGetEntity),
-		rest.Get("/ngsi-ld/v1/entities", apisrv.GetQueryParamsEntities),
-
-		//delete
 		rest.Delete("/ngsi-ld/v1/entities/#eid", apisrv.DeleteLDEntity),
 		rest.Delete("/ngsi-ld/v1/entities/#eid/attrs/#attr", broker.LDDeleteEntityAttribute),
 
-		//subscription
 		rest.Post("/ngsi-ld/v1/subscriptions/", broker.LDCreateSubscription),
 		rest.Get("/ngsi-ld/v1/subscriptions/", broker.GetLDSubscriptions),
 		rest.Get("/ngsi-ld/v1/subscriptions/#sid", apisrv.GetLDSubscription),
 		rest.Patch("/ngsi-ld/v1/subscriptions/#sid", broker.UpdateLDSubscription),
 		rest.Delete("/ngsi-ld/v1/subscriptions/#sid", apisrv.DeleteLDSubscription),
-
-		//notify
```

Comment:
this should not be deleted, its comments which tells about the below API

Reply:
Done in other pull request